### PR TITLE
Create tanishqamitra.json

### PR DESCRIPTION
**Pull request by tanishqamitra**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7016"></a>

### CODE REVIEW

File: tanishqamitra.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Tanishqa Mitra",
+  "type": "personal",
+  "bio": "Open Source 🥑 •  Co-founder at @zeal_yug •   Join me on a journey of constant learning & growth!"
```
Comment (SaraJaoude) [inline tanishqamitra.json:4]: add a comma:
```suggestion
  "bio": "Open Source 🥑 •  Co-founder at @zeal_yug •   Join me on a journey of constant learning & growth!",
```

File: tanishqamitra.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/MitraTanishqa",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/tanishqa-mitra/",
+      "icon": "FaInstagram"
+    }
```
Comment (SaraJaoude) [inline tanishqamitra.json:15]: add a comma:
```suggestion
    },
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder. 

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

File: tanishqamitra.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Tanishqa Mitra",
+  "type": "personal",
```
Comment (kumarsonsoff3) [inline tanishqamitra.json:3]: Also, please remove the `type` key from your `json`.
```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Hey everyone,

Thank you all for the review. Since the pull request has been open for a while, I will be merging it into a temporary branch to address any existing errors before merging it into the main branch.

**Pull request merged**